integration-tests/ci-visibility/test-optimization-wrong-init.spec.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+const { exec } = require('child_process')
+
+const { assert } = require('chai')
```

Comment:
Could we use Node.js assert instead? :)

We recently decided in the guild to slowly move to those and introducing more chai assertions makes it harder.